requirements.txt

```diff
@@ -0,0 +1,2 @@
+# generated from manifests external_dependencies
+odoo_test_helper
```

Comment:
Could you please cherry-pick https://github.com/OCA/bank-statement-import/commit/9df67995671b2f47b91d591e47ebf38af3115f1c? :)

Reply:
Done!